### PR TITLE
Edits to Migration Portal: added unsupported objects to known issues …

### PR DESCRIPTION
**Pull request by ebgitelman**
…PR5943

### CODE REVIEW

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx [hunk 1]
```diff
@@ -12,7 +12,7 @@ legacyRedirectsGenerated:
 You can assess an Oracle database schema for compatibility with EDB Postgres Advanced Server. You can also generate a report about the assessment.
 
 !!!note 
-   Review the [release notes](../01_mp_release_notes/) and [known issues](../known_issues_notes.mdx) section for an overview of unsupported schema objects.
+   Review the [release notes](../01_mp_release_notes/) and [known issues](../known_issues_notes.mdx) for an overview of unsupported schema objects.
```
Comment (ebgitelman) [inline product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx:15]: @gvasquezvargas  I have a query in the next procedure. Why is there a screen capture of the New Project dialog box? We don't include screen captures of dialog boxes unless they are filled in in a certain way that is instructive.

File: product_docs/docs/migration_portal/4/known_issues_notes.mdx [hunk 2]
```diff
@@ -23,15 +23,15 @@ Some unsupported Oracle objects are removed when Migration Portal assesses your
...
-  EDB only supports granting `CREATE DATABASE LINK`, `CREATE PUBLIC DATABASE LINK`, `DROP PUBLIC DATABASE LINK` and `EXEMPT ACCESS POLICY` system privileges. Other `GRANT` statements are not supported and removed from the DDL file. 
+- Objects related to `MATERIALIZED VIEWS`, for example, backend `TABLE` or `INDEX` statements created to support `MVIEW`
+- Objects related to `Queues`
+- Objects related to `Nested Tables`
+- Objects related to `XMLType Tables`
+- Types that depend on `SYSTEM Schemas`
+- Indexes related to `PRIMARY KEY` and `UNIQUE` constraints
+- Unsupported system `GRANT` privileges
+
+  EDB supports granting only `CREATE DATABASE LINK`, `CREATE PUBLIC DATABASE LINK`, `DROP PUBLIC DATABASE LINK`, and `EXEMPT ACCESS POLICY` system privileges. Other `GRANT` statements aren't supported and are removed from the DDL file.
```
Comment (ebgitelman) [inline product_docs/docs/migration_portal/4/known_issues_notes.mdx:34]: It looked like the word "are" was missing from the original. Is my edit correct?

Comment (gvasquezvargas) [reply product_docs/docs/migration_portal/4/known_issues_notes.mdx:34]: You are correct, good catch!

File: product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx (hunk 1 above)
Comment (gvasquezvargas) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx:15]: I am not sure, as the screenshots are before my time. If you believe we should rethink how we use screenshots on this page, let's sync and we can pitch something to the PM/team.

Comment (ebgitelman) [reply product_docs/docs/migration_portal/4/04_mp_migrating_database/02_mp_schema_assessment.mdx:15]: It's not really something we need to pitch. We don't include dialog boxes as a rule. It looks like you added it a few months ago. I'm going to remove it.

**Review by gvasquezvargas — APPROVED** [review]
LGTM!

**Pull request merged**